src/BlazorUI/Bit.BlazorUI/Components/Surfaces/ScrollablePane/BitScrollablePane.scss

```
@@ -2,34 +2,226 @@
     }
 
     &::-webkit-scrollbar-button {
         display: none;
     }
 }
+
+// The overlay behavior of a modern operating system, for the scrollbar the theme draws: the thumb is
+// only painted while the pointer is over the pane, while something inside it holds the focus, or while
+// the pane itself does - so it is never hidden from a reader who is scrolling it with the keyboard.
+// Only the color goes away: the bar keeps its room, so nothing in the pane moves as it comes and goes.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)